src/firebolt/service/engine.py

```diff
@@ -186,11 +188,8 @@ def create(
             specification=EngineRevisionSpecification(
                 db_compute_instances_type_key=instance_type_key,
                 db_compute_instances_count=scale,
-                db_compute_instances_use_spot=False,
```

Comment:
Do we wan't to have some defaults for `db_compute_instances_use_spot`, `db_version`, `proxy_instances_count` and `proxy_version`?

Reply:
We already have those defaults in `EngineRevisionSpecification`, https://github.com/firebolt-db/firebolt-python-sdk/blob/edf5df76ad59ce0192c3c34ff9816542d0621884/src/firebolt/model/engine_revision.py#L34